internal/yaml.go

```diff
@@ -0,0 +1,162 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package internal
+
+import (
+	"fmt"
+
+	"golang.org/x/exp/maps"
+)
+
+// Stats represent the expected/actual amount of
+// failed, skipped and passed tests.
+type Stats struct {
+	UnexpectedRest int `yaml:"unexpected_rest"`
+	UnexpectedFail int `yaml:"unexpected_fail"`
+	UnexpectedSkip int `yaml:"unexpected_skip"`
+	UnexpectedPass int `yaml:"unexpected_pass"`
+	ExpectedFail   int `yaml:"expected_fail"`
+	ExpectedSkip   int `yaml:"expected_skip"`
+	ExpectedPass   int `yaml:"expected_pass"`
+}
+
+// Structs and their internal equivalents:
+// - ConfigYAML -> Config
+// - ResultsYAML -> Results
+// - TestsConfigYAML -> TestsConfig.
+
```

Comment:
This empty line will divide the doc into 2 separate parts, do we really want this? 
Let's maybe add a `//` here.
```suggestion
//
```

Reply:
I think they should be seperated. When we add `//` this comment will be added specifically to the `ConfigYAML` struct (and not to `ResultYAML` and `TestsConfigYAML`).